### DISCUSSION TITLE
Grupo N_5 - My PyC - Tarea T1 - CTY

### DISCUSSION THREAD

**Discussion by Acentor** [santiemanuel/milProgramadoresQA#29] [Simulacro Trabajo Grupal]
**Hola presento lo que fuimos haciendo, lo planteamos en un diagrama de flujo** 

![Blank diagram](https://user-images.githubusercontent.com/102877482/198892338-c0a26f27-4cda-4682-bbbc-c9b02ddaf2a1.png)

**se usa la placa madre como punto de partida y de allí se verifica los componentes, al ser la placa madre la que impone en su mayoría el limite de compatibilidades**

**También hicimos otro diagrama sobre la relación de dependencia**

https://lucid.app/lucidchart/39c05997-7e83-436b-a884-7c45167609ac/edit?invitationId=inv_66c72cb3-10fe-4c5a-a27e-2c60bf0b58ba&page=HWEp-vi-RSFO#

**Comment by Acentor** [comment]
![Imagen de WhatsApp 2022-10-24 a las 21 28 19](https://user-images.githubusercontent.com/102877482/198741778-d0358a01-77b4-4f09-9f04-b8a28f2be6b5.jpg)

**Reply by santiemanuel** [reply]
Buenas noches, el flujo sí les servirá para la próxima etapa. Aquí lo que podría notar es que la placa de sonido viene integrada siempre en las placas base actuales.

Respecto a la RAM, se puede limitar a los MHz soportados entre CPU y RAM, si el CPU soporta 2000 MHz y la RAM instalada es de 3200 MHz, entonces solo la limitamos a 2000, no decimos que no se puede instalar. Quito la parte de compatiblidad con la placa base porque entre 3 componentes se complejizaría la comprobación.

Para SATA, también simplificar a solo un tamaño así que sería un atributo menos, y solo limitar a bahías disponibles en el gabinete. También para almacenamiento limitamos a puertos SATA y ranuras M.2 para reducir la complejidad.

Tipo de discos también, dejamos entre SATA y NVME, los IDE ya están obsoletos y nos ayuda a reducir la complejidad del modelo.

La compatibilidad de PCIe, como la RAM, no es absoluta. Es equivalente, es decir, una GPU que soporta PCIe 5.0 puede funcionar en uno 3.0 pero se verá limitado en el ancho de banda disponible.

Y con eso podrían limpiar un poco las condiciones que son demasiado específicas o ya obsoletas en cuanto a tecnología.

**Comment by Acentor** [comment]
![Blank diagram0 2](https://user-images.githubusercontent.com/102877482/199064263-cf8cff4e-02e9-4568-b162-8cd8bde8ba6c.png)

creo que asi serian los cambios, espero haber comprendido, al menos esta mas corto, lo de pcie y ram solo los puse entre paréntesis para enumerar, tenia un apartado de eficiencia, pero no quería agregar mas
